### DISCUSSION TITLE
Lesson 6: compiling error "ConnectionError: Status 404 when getting package versions from Github'"

### DISCUSSION THREAD

**Discussion by christianag** [smartcontractkit/full-blockchain-solidity-course-py#566] [Q&A] [answered]
Hey everyone,

First - a big thank you for all of the hard work that went into creating this amazing 16+ hour lesson. Truly grateful for all of your hard work.

At around 05:10:00 when we're supposed to **_brownie compile_** for the FundMe file, I get the following error:

```
ConnectionError: Status 404 when getting package versions from Github: 'Not Found'
Missing or forbidden.
If this issue persists, generate a Github API token and store it as the environment variable `GITHUB_TOKEN`:
https://github.blog/2013-05-16-personal-api-tokens/
```

![compile_error](https://user-images.githubusercontent.com/29675397/146040636-9bba0291-3351-4205-bee2-e9a568ee0d52.png)

Here is my config file:
![compile2](https://user-images.githubusercontent.com/29675397/146041178-ce354c6e-6e14-4c1b-b267-40993714f9c4.png)

I'll be very grateful to anyone who has any ideas how to solve this. Thank you!

**Comment by ZeroEkkusu** [comment, marked as answer]
A typo
Should be `smartcontractkit/chainlink-brownie-contracts@1.1.1`

**Comment by christianag** [comment]
Amazing catch! 
It works now, thank you so much for the help :)
